### ISSUE TITLE
Allow debug signing (so keystore config isn't needed)

### ISSUE THREAD

**Issue by posti85** [vaibhavpandeyvpz/apkstudio#143]
The version 4 of Apkstudio allowed debug signing, so you don't need to add and configure the android keystore file in the settings. It could be great to have a checkbox in the signing config screen to tell if you want debug signing, so the keystore config fields would turn disabled due they are not needed.

**Comment by vaibhavpandeyvpz** [comment]
It’s there in this one as well. You can just skip keystore config in APK Studio and hit sign. It should debug sign it.

**Comment by posti85** [comment]
I didn't know it. Thank you very much! 

I have to say that it was hard to realize that the APK signing options in the toolbar are enabled when the APK file is highlighted. When I saw them disabled after the first time of building, I thought I had to fill the signing configuration to enable them, but the clue was on highlighting the APK file. I think it is a usability issue: APK studio should sign the existing built file without the need of highlighting it.
